### PR TITLE
Added editor.fullScreenOnStart functionality to Config file.

### PR DESCRIPTION
**Pull request by bert88sta**
I don't exactly know why the commits for fixing #196 are listed here, but the changes in index.tsx and Config.ts enable fullscreen startup

### CODE REVIEW

File: browser/src/index.tsx [hunk 1]
```diff
@@ -26,14 +26,16 @@ import { OverlayManager } from "./UI/Overlay/OverlayManager"
...
     const services: any[] = []
 
     const parsedArgs = minimist(args)
     const debugPlugin = parsedArgs["debugPlugin"] // tslint:disable-line no-string-literal
 
     // Helper for debugging:
     window["UI"] = UI // tslint:disable-line no-string-literal
-
+    remote.BrowserWindow.getFocusedWindow().setFullScreen(Config.getValue<boolean>("editor.fullScreenOnStart"))
+    // remote.BrowserWindow.getFocusedWindow().setFullScreen(true)
```
Comment (extr0py) [inline browser/src/index.tsx]: Can you please remove this extra commented line?

File: browser/src/index.tsx [hunk 2]
```diff
@@ -26,14 +26,16 @@ import { OverlayManager } from "./UI/Overlay/OverlayManager"
...
+
     const services: any[] = []
 
     const parsedArgs = minimist(args)
     const debugPlugin = parsedArgs["debugPlugin"] // tslint:disable-line no-string-literal
 
     // Helper for debugging:
     window["UI"] = UI // tslint:disable-line no-string-literal
-
+    remote.BrowserWindow.getFocusedWindow().setFullScreen(Config.getValue<boolean>("editor.fullScreenOnStart"))
```
Comment (extr0py) [inline browser/src/index.tsx]: Does setting this to `false` have any behavioral change than just not calling it at at all? Just curious!

File: main.js [hunk 3]
```diff
@@ -78,7 +78,7 @@ function createWindow(commandLineArguments, workingDirectory) {
        {
            label: 'Redo',
            click: (item, focusedWindow) => {
-               mainWindow.webContents.send("menu-item-click", "\<C-r>")
+               mainWindow.webContents.send("menu-item-click", "\\<C-r>")
```
Comment (extr0py) [inline main.js]: Looks like the changes from the menu change slipped in here too :) You might want to pull the latest master and merge or rebase those changes in.

**Comment by extr0py** [comment]
That's great that you're getting your feet wet in the typescript world :) Overall the change looks good, I just had some minor comments. It looks like the build is failing as well - seems like a missing comma / lint issue in Config.ts. Thanks for the PR!

File: browser/src/index.tsx (hunk 2 above)
Comment (bert88sta) [inline browser/src/index.tsx]: Allegedly it changes the ability to fullscreen the window, but I think that's only when passed as an initial parameter. I was also wanting to look into this vs. maximize(), but fullscreen is much easier to implement for now.

Comment (bert88sta) [inline browser/src/index.tsx]: Also worth mentioning that I can still fullscreen the window even with it set to false on OS X

**Comment by bert88sta** [comment]
@extropy: I fetched upstream and merged in (glad to finally learn how git actually works). In the 'compare' section of the future PR, it doesn't show the diff of  the copy-paste-redo-undo menu, but the commits are still listed.... I'm assuming I have to rebase but I've no clue what or how that works

File: browser/src/index.tsx (hunk 2 above)
Comment (extr0py) [inline browser/src/index.tsx]: Cool, thanks for looking into it!

**Comment by extr0py** [comment]
@bert88sta - great, the diff looks good now! As long as the 'compare' section looks good then we're set. You could potentially 'rebase', which lets you pick and choose commits to be applied on top of master, to get rid of those old ones - but I'll do a squash merge here, which will combine all the commits into a single commit and bring into master. Git can be pretty intense, especially at first :)

Thanks for the contribution!

**Pull request closed without merging**